### ISSUE TITLE
How to get the gradient of a DCCP problem

### ISSUE THREAD

**Issue by lorenzosteccanella** [cvxgrp/dccp#72]
Is it possible to get the gradient of a DCCP as you would do with DPP problem with requires_grad=True and backward?

Thanks in advance for the help.

**Comment by xinyueshen** [comment]
Not sure what you mean by gradient of a DCCP, but you can get the gradient of a DCP expression from the `.grad` attribute. Here is an example.
```
x = cvxpy.Variable()
x.value = 2
expr = cvxpy.exp(x)
print(expr.grad[x]) # gradient at x=2
```

**Comment by lorenzosteccanella** [comment]
I was thinking more at something like as you would do for the parameters of a DPP or with cvxpylayer:

import cvxpy as cp

x = cp.Variable()
p = cp.Parameter()
quadratic = cp.square(x - 2 * p)
problem = cp.Problem(cp.Minimize(quadratic))
p.value = 3.
problem.solve(requires_grad=True)
problem.backward()
print("The gradient is {0:0.1f}.".format(p.gradient))

**Comment by SteveDiamond** [comment]
@lorenzosteccanella this unfortunately is not supported.
